### ISSUE TITLE
Scan reports missing Expect-CT header although it is being set

### ISSUE THREAD

**Issue by peteeveleigh** [putyourlightson/craft-sherlock#32]
### Describe the bug
Running a Sherlock Scan reports a warning about the Expect-CT header not being set. However it is being set (from with the Sherlock plugin) and appears in the headers. A scan performed by an external service confirms the header is present and apparently valid.

I was previously setting this header via htaccess and Sherlock didn't indicate that it was missing.

### To reproduce
Set an Expect-CT header using the Security Headers feature within Sherlock, then perform a scan.

### Expected behaviour
The Expect-CT headers should get a green tick as it's present and no warning raised.

### Screenshots
Screenshots showing output from Sherlock and the Response Headers from the browser console.

![Screenshot 2021-07-15 at 11 23 07](https://user-images.githubusercontent.com/827065/125773649-6450e1c1-09d1-4658-b8a2-dfdbea39d574.png)
![Screenshot 2021-07-15 at 11 26 12](https://user-images.githubusercontent.com/827065/125773651-35394726-ecd5-4bc9-b43b-5f479be0e83e.png)

We are slightly behind with updates so this may have already been addressed. Hoping to get the update applied this week.

### Versions
- Plugin version: Lite 3.1.2
- Craft version: Pro 3.6.11.2

**Comment by peteeveleigh** [comment]
I've also tried setting the header via htaccess *with* it left in the Sherlock settings too. And the same issue applies.

Only if I remove the Expect-CT header from the Sherlock settings does the scan then pick it up.

**Comment by bencroker** [comment]
Are you using a CDN to serve the site? If so then it may be worth purging the cache as it is possible that the response you are seeing is not the same as the response that Sherlock is (requested from your web server).

**Comment by peteeveleigh** [comment]
Hi Ben
No, it's hosted on a Krystal shared hosting account. The behaviour is really weird to be honest.
I spent a long time today setting up the CSP and there were some caching issues with that in Chrome. Everything seems good now though (needs a browser restart to properly flush the CSP out) but the Expect-CT header just behaves in this weird way.

I've added that one header via htaccess for the moment and turned it of in Sherlock. Will do some more investigations but don't really know where to go with it.

Pete
